### PR TITLE
Kld/master

### PR DESCRIPTION
**Pull request by kdefliese**
Finished Wave 1

### CODE REVIEW

File: views/about_me.erb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<html>
```
Comment (kariabancroft) [inline views/about_me.erb:1]: Watch out for indentation, once you open a tag and start nesting, each subsequent tag should be indented

File: views/about_me.erb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+<head>
+  <title>About Me</title>
+</head>
+<header>
+  <h3><a href="index.html" />Home</a>   <a href="blog.html" />Blog</a>   <a href= "projects.html" />Projects</a></h3>
```
Comment (kariabancroft) [inline views/about_me.erb]: Is there a way to accomplish the same layout but have each `a` tag on a separate line?

File: views/blog.erb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+</head>
+<header>
+  <h3><a href="index.html" />Home</a>   <a href="about_me.html" />About Me</a>   <a href= "projects.html" />Projects</a></h3>
+</header>
+<section>
+  <h2>
+    October 26, 2015: My favorite real-life cat!
+  </h2>
+  <p>
+    This is a blog post about my favorite cat, Maxwell. Here is a picture of me with my favorite cat. He is fat.
```
Comment (kariabancroft) [inline views/blog.erb]: Hello Maxwell

File: views/about_me.erb (hunk 1 above)
Comment (kdefliese) [inline views/about_me.erb:1]: Thanks, fixed it!

File: views/blog.erb (hunk 3 above)
Comment (kdefliese) [inline views/blog.erb]: 😸

File: views/about_me.erb (hunk 2 above)
Comment (kdefliese) [inline views/about_me.erb]: Do you mean like this? (took out actual tags so it wouldn't convert the html)
h3
      a href="about_me.html" />About Me
      a href= "projects.html" />Projects
      a href="blog.html" />Blog
    /h3

**Pull request merged**